alembic/ddl/mysql.py

```diff
@@ -391,6 +393,29 @@ def _mysql_alter_default(
     )
 
 
+@compiles(AddColumn, "mysql", "mariadb")
+def _mysql_add_column(
+    element: AddColumn, compiler: MySQLDDLCompiler, **kw
+) -> str:
+
+    return "%s ADD COLUMN %s%s" % (
```

Comment:
how come this is done so differently vs. the postgresql version?

Reply:
Main element of explanation: I privileged local style consistency, and the implem files are way different one from the other. Also I did postgresql way before and worked on this file at another moment, hence privileging local style over daring to harmonise the contribution and make this change seem odd compared to the rest of the file